radix-engine/src/updates/cuttlefish.rs

```diff
@@ -466,3 +478,121 @@ fn generate_cuttlefish_update_min_rounds_per_epoch<S: SubstateDatabase + ?Sized>
         consensus_manager_config.into_locked_substate(),
     )
 }
+
+/// A quick macro for encoding and unwrapping.
+macro_rules! scrypto_encode {
+    (
+        $expr: expr
+    ) => {
+        ::radix_common::prelude::scrypto_encode($expr).unwrap()
+    };
+}
+
+fn generate_cuttlefish_update_identity_to_not_create_royalty_module<
```

Comment:
Very minor: for code consistency, maybe we should refactor this to be similar to our new API for constructing state updates: https://github.com/radixdlt/radixdlt-scrypto/blob/feature/account-getters/radix-engine/src/updates/cuttlefish.rs#L223-L360

Reply:
May as well fix this when we merge develop :)